### ISSUE TITLE
Error when using this role with Ansible

### ISSUE THREAD

**Issue by back-2-95** [andrewrothstein/ansible-docker#1]
ERROR: package is not a legal parameter in an Ansible task or handler
Ansible failed to complete successfully. Any error output should be
visible above. Please fix these errors and try again.

I'm using Vagrant box geerlingguy/centos7. Role was working about a week ago.

**Comment by andrewrothstein** [comment]
I upgraded the role to take advantage of the package module in Ansible 2. Recently switched up to Fedora 23 from 21. Rather than adding in more tasks with when: clauses for dnf I gave up support for Ansible 1.x. I highly recommend upgrading. Apologies if this stopped working for you.

**Comment by back-2-95** [comment]
Hi, Good to know. I'll try Ansible 2.

Maybe stating that in the readme.md as a requirement?

**Comment by andrewrothstein** [comment]
I marked it as requiring ansible 2 in the role metadata https://github.com/andrewrothstein/ansible-docker/blob/master/meta/main.yml. Apologies if you were looking on the galaxy. I've not been #disciplined about re-importing the roles into the galaxy when I change their dependencies or minimum required versions. I reimported it just now and it reflects the appropriate minimum version. Thanks for pointing that out. My documentation could use a big improvement. Don't be shy with the pull requests :-) Regards, Andrew
